roles/rsyslog/tasks/main.yml

```diff
@@ -54,6 +54,15 @@
         - rsyslog_unprivileged | bool
         - rsyslog_user != 'root'
 
+    - name: Create the config directory if it does not exist
+      file:
+        state: directory
+        path: '{{ rsyslog_config_dir }}'
+        owner: '{{ rsyslog_user }}'
+        group: '{{ rsyslog_file_group }}'
+        mode: 0755
+      changed_when: false
```

Comment:
Why use `changed_when: false` here?

Reply:
A good question...  I copied an existing directory creation task.  I found this comment [1].  I guess we should consider "change" any changes made to the host, not just to the rsyslog configuration?  If so, should we remove changed_when in the file/directory creation tasks?  (I remove this "changed_when: false" in this PR with opening another for the rest.)
[1]
```
commit 85253790ea38f04829e68255353c0335d8165e27
Author: Shirly Radco <sradco@redhat.com>
Date:   Mon Jan 28 16:44:52 2019 +0200
    Updated backup task - added changed_when as False,
    Since the task does not actually changes Rsyslog configurations.
    Signed Of By: Shirly Radco
```

Reply:
Hmm - not sure what Shirly was doing there - but in general, you want to know when something changed - you want to know if the directory was created

Also, if you used `changed_when: false` in other places, yes, we should review them, in another PR

Reply:
Ah, she put `changed_when: false` when creating a work dir `/var/lib/rsyslog` and a back up dir `/tmp/rsyslog.d-DATETIME` since they are not /etc/rsyslog.conf nor /etc/rsyslog.d related.  Probably, we could get rid of them?

Reply:
>  Probably, we could get rid of them?

yes